src/holofuel/pages/CreateOffer/CreateOffer.test.js

```diff
@@ -235,6 +237,57 @@ describe('CreateOffer', () => {
     expect(within(getByTestId('counterparty-nickname')).getByText('Loading')).toBeInTheDocument()
   })
 
+  it('renders a clickable list of recent counterparties', async () => {
+    const agent1 = {
+      id: 'fkljd',
+      nickname: 'Jo'
+    }
+
+    const agent2 = {
+      id: 'dskajln',
+      nickname: 'Bob'
+    }
+
+    const mocks = [
+      {
+        request: {
+          query: HolofuelCounterpartyQuery,
+          variables: { agentId: agent1.id }
+        },
+        result: {
+          data: { holofuelCounterparty: agent1 }
+        }
+      },
```

Comment:
Do we need the HolofuelCounterpartyQuery mock here?  If not, can we remove it?

Reply:
Good catch, removed.